vertex_launcher/src-tauri/src/commands.rs

```diff
@@ -226,6 +226,10 @@ pub async fn download(app_handle: tauri::AppHandle, game: u8) -> errors::Result<
             env::STORE_LOCAL_GAME_LIST_KEY,
             serde_json::to_value(&*game_list).unwrap(),
         );
+        app_handle.emit(
+            &format!("download_completed_{}", game),
```

Comment:
Prefer using the constant event name from `src-tauri/src/env.rs` for better maintainability.
```suggestion
            &format!("{}_{}", env::CONST_EVENT_NAME_FROM_ENV, game),
```